tests/integration/test_multiple_response.py

```diff
@@ -375,11 +375,58 @@ def test_various_measures_from_r_rows_margin():
         ],
     ]
 
+    expected_col_moe = [
```

Comment:
all of the expectations for this test should be extracted. Together they are like three screenfuls of noise in the test. We're doing this incrementally as we encounter legacy tests like this one.